clover/src/optical_flow.cpp

```diff
@@ -112,9 +114,31 @@ class OpticalFlow : public nodelet::Nodelet
 
 		auto img = cv_bridge::toCvShare(msg, "mono8")->image;
 
-		// Apply ROI
-		if (roi_ != 0) {
-			img = img(cv::Rect((msg->width / 2 - roi_2_), (msg->height / 2 - roi_2_), roi_, roi_));
+		if (roi_.empty()) {
```

Comment:
`emtpy()` is defined as `width <= 0 || height <= 0` (https://github.com/opencv/opencv/blob/master/modules/core/include/opencv2/core/types.hpp#L1898-L1902)

Reply:
Done, waiting for tests...

Reply:
Native tests passed